docs/learn/bitesize-invaders.md

```diff
@@ -0,0 +1,309 @@
+```
+
+<!-- ```csharp reference
+https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/Basic/Invaders/Assets/Scripts/LobbyControl.cs#L158-L186
+``` -->
+
+## Unconventional Networked Movement
+
+Invaders has an easy movement type - moving only on one (horizontal) axis - which allows you to only modify the transform client-side. Modify *[PlayerControl.cs](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/Basic/Invaders/Assets/Scripts/PlayerControl.cs)* in the `InGameUpdate` function with the help of a [`NetworkTransform`](../components/networktransform.md) that is attached directly to the Player Game Object and automatically syncs up the Transform with the other clients. It will smooth out by interpolating or extrapolating for all of them.
+
+```csharp
+private void InGameUpdate()
+{
+    if (!IsLocalPlayer || !IsOwner || !m_HasGameStarted) return;
+    if (!m_IsAlive) return;
+
+    var deltaX = 0;
+    if (Input.GetKey(KeyCode.LeftArrow)) deltaX -= 1;
+    if (Input.GetKey(KeyCode.RightArrow)) deltaX += 1;
+
+    if (deltaX != 0)
+    {
+        var newMovement = new Vector3(deltaX, 0, 0);
+        transform.position = Vector3.MoveTowards(transform.position, transform.position + newMovement, m_MoveSpeed * Time.deltaTime);
+    }
+
+    if (Input.GetKey(KeyCode.Space)) ShootServerRPC();
+}
+```
+
+<!-- ```csharp reference
+https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/Basic/Invaders/Assets/Scripts/PlayerControl.cs#L157-L173
+``` -->
+
+## Shared Start/Round Timer with Client-Side Prediction
+
+Games commonly have timers to display in the UI such as Start Timer, Round Timer, and Cooldowns. Invaders also has a shared timer to ensure all players start the game at the same time. Otherwise, players with higher end devices and better network access may have an unfair advantage by loading scenes and maps faster.
+
+When you implement this kind of timer, usually you would use a `NetworkVariableFloat` to replicate and display the exact time value across all clients. To improve performance, you do not need to replicate that float every Network Tick to the Clients, which would only waste network bandwith and some minimal CPU resources.
+
+You have two options:
+
+* The best solution is to sync the remaining value of the timer when a new client joins. For the remaining time, clients can locally predict what the next value of that timer is going to be. This method ensures the server does not need to send the value of that timer every Network Update tick since you know what the approximated value will be. There is a minimal overhead of keeping an additional float member variable that will be kept updated, as the clients are not able to modify the `NetworkVariable` directly.
+* A fair solution is to set the `SendTickRate` of that timer `NetworkVariableFloat`, so that the server only sends an update once every second, without no additional work.
```

Comment:
```suggestion
* A fair solution is to set the `SendTickRate` of that timer `NetworkVariableFloat`, so that the server only sends an update once every second, without any additional work.
```